.github/workflows/stale.yaml

```
@@ -0,0 +1,34 @@
+name: 'Close stale'
+
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+permissions:
```

here should be, this permission should be set in the job level

```suggestion
permissions: {}
```